include/zephyr/dt-bindings/pwm/it8xxx2_pwm.h

```diff
@@ -21,4 +21,15 @@
 #define PWM_CHANNEL_6		6
 #define PWM_CHANNEL_7		7
 
+/*
+ * Provides a type to hold PWM configuration flags.
+ *
+ * The upper 8 bits are reserved for SoC specific flags.
+ *    Output onpe-drain flag    [ 8 ]
+ */
+#define PWM_IT8XXX2_FLAGS_SHIFT		8U
+
+#define PWM_IT8XXX2_OPEN_DRAIN		(1 << PWM_IT8XXX2_FLAGS_SHIFT)
```

Comment:
You could include `#include <zephyr/dt-bindings/dt-util.h>` use `BIT(PWM_IT8XXX2_FLAGS_SHIFT)` if you fancy it.

Reply:
...or even just `BIT(8)` at that point, if you do the other change you don't need the shift define for anything else anyway.

Reply:
Done.